db/models.py

```diff
@@ -1 +1,30 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255, help_text="skill bonus type")
+    race = models.ForeignKey(Race, on_delete=models.SET_NULL, null=True)
```

Comment:
I think it's better to use on_delete=models.CASCADE, without null=True, because we don't need skills that exist without race